lab_5_scrapper/scrapper.py

```diff
@@ -26,11 +38,36 @@ def _extract_config_content(self) -> ConfigDTO:
         Returns:
             ConfigDTO: Config values
         """
+        with open(self.path_to_config) as config_file:
+            config = json.load(config_file)
+        return ConfigDTO(seed_urls=config['seed_urls'],
+                         total_articles_to_find_and_parse=config['total_articles_to_find_and_parse'],
+                         headers=config['headers'],
+                         encoding=config['encoding'],
+                         timeout=config['timeout'],
+                         should_verify_certificate=config['should_verify_certificate'],
+                         headless_mode=config['headless_mode']
+                         )
 
     def _validate_config_content(self) -> None:
         """
         Ensure configuration parameters are not corrupt.
         """
+        config = self._extract_config_content()
```

Comment:
why do you do it twice? first DTO is created here and then again in init

Reply:
fixed